src/lib/auth.ts

```diff
@@ -56,7 +59,13 @@ export const authOptions: NextAuthOptions = {
           image: castedProfile.avatar_url,
         })
 
-        // TODO(Agnieszka): Send email to user about successful registration
+        if (createdUser) {
+          await mailersendClient.sendMail(
+            [new Recipient(createdUser.email, castedProfile.login)],
+            process.env.MAILERSEND_TEMPLATE_ID!,
```

Comment:
**nitpick:** We should not keep TEMPLATE_ID, and SEND_SUBJECT in `.env` because we will have a lot of them, and currently it suggests that we have only one. We should create something like, which will be held somewhere in `mailerSendClient.ts`:

```ts
enum TransactionMailTemplateIds {
 welcomeMail = "12355asdgg",
 // then, we will add more template ids later
 ...
}
```

And then:

```ts
 await mailersendClient.sendMail(
            [new Recipient(createdUser.email, castedProfile.login)],
            TransactionMailTemplateIds.welcomeMail,
            "Welcome to Good Dev!",
```